### PR TITLE
feat(app): #17 add redeploy hook

### LINKED ISSUE #17: Trigger rebuild when working in CMS

### PR DESCRIPTION
**Pull request by MrGreenTea**
and use it when Events are changed

Closes #17

### CODE REVIEW

File: src/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const redeploy = async () => {
+  await fetch(
+    "https://api.cloudflare.com/client/v4/pages/webhooks/deploy_hooks/5d30d67d-725a-4c5f-9b60-83f1a457ac9d",
```
Comment (Han2-Ro) [inline src/hooks.ts:3]: This url should probably not be in the public repo. Use a environment variable instead.

**Review by Han2-Ro — CHANGES_REQUESTED** [review]

File: src/hooks.ts (hunk 1 above)
Comment (MrGreenTea) [reply src/hooks.ts]: I have added the (new) deploy hook to coolify:
https://coolify.youngvision.work/project/t04c44g/production/application/v8kgggw#environment-variables

**Comment by MrGreenTea** [comment]
We'll have to make sure that we only redeploy when changes are published, and not for drafts

**Review by Han2-Ro — APPROVED** [review]
Nice

**Pull request merged**